Design history entry: exploring how schools tell us about teachers leaving (#270)
DFE-Digital/teacher-cpd-design-history: app/posts/manage-training/2023-11-08-teachers-leaving-schools.md

Reviewer: All done Frankie. Very comprehensive and I don't have anything to add to the research part, its all covered.

---
title: Exploring how schools tell us when a teacher has left
description: We removed an explicit option for induction tutors to add themselves as a mentor, instead allowing them to do it through the normal “add mentor” flow.
date: 2023-11-08
tags:
  - early career teachers
image:
  src: /manage-training/teachers-leaving-schools/teachers-leaving-schools.png

>   alt: "Illustration showing the question “Is Jonathan left the school permanently?” with the answers “Yes” or “No, they’ve taken a temporary leave of absence, for example parental leave” as radio options"

Reviewer 2: ```suggestion
  alt: "Illustration showing the question “Has Jonathan left the school permanently?” with the answers “Yes” or “No, they’ve taken a temporary leave of absence, for example parental leave” as radio options"
```

  opengraphImage: true
---

We did some design and research to better understand how and when schools would tell us when one of their early career teachers leaves their school.

Our research showed that schools did not always understand when they were supposed to do this.

## Current process

> Schools using the service see a link on bottom of the page for each early career teacher with the text ‘Tell us if (teacher’s name) is transferring to another school’.

Reviewer 2: ```suggestion
Schools using the service see a link at the bottom of the profile page for each early career teacher with the text ‘Tell us if (teacher’s name) is transferring to another school’.
```

![TODO](current-leaving-link.png)

Clicking this takes them to a confirmation page:

![TODO](current-confirmation-page.png)

> If they continue, they are then asked for a date when the teacher left the school. This can be in the past or future:

Reviewer 2: ```suggestion
If they continue, they are then asked for a date when the teacher has or will leave the school. This can be in the past or future:
```

The initial link was reworded to ‘Report that (teacher’s name) is leaving your school or taking a leave of absence’.

![TODO](new-leaving-link.png)

The next page then asked for their leaving date:

![TODO](new-date.png)

We then ask whether the teacher is leaving the school permanently or not. The question is worded using past or future tense based upon the leaving date.

![TODO](new-leaving-permanently.png)

If they are leaving the school permanently, we ask whether they are moving to another school in England. Schools can answer ‘Yes’, ‘No’ or ‘I do not know’.

If they select ‘Yes’, they are asked which school the teacher is moving to. This uses an autocomplete.

![TODO](new-moving-to-another-school.png)

They are then shown a check your answers page, as before:

![TODO](new-check-answers.png)

The confirmation page was updated to be clearer about what schools need to do next, by including the names of the appropriate body and training provider:

![TODO](new-completion-page.png)

When the school returns to their list of early career teachers, any teachers who have left their school are shown along with the date that teacher left:

![TODO](new-list-view.png)

> Teachers who are leaving in the future are shown as currently training, but with the date they are leaving at shown:

Reviewer 2: ```suggestion
Teachers who are leaving in the future are shown as currently training, but their reported leaving date will also be shown:
```

![TODO](new-leaving-in-future.png)

## The research

We ran 2 focus group sessions with school induction tutors from 6 schools (2 secondary, 4 primary).

These participants were invited from a sample of schools who had previously had early career teachers leave during their induction.

> In the focus groups, the participants were asked about their experience of teachers leaving school during their early career induction. They were also shown the new prototype design and asked to use it to tell us that one of the teacher was leaving the school.

Reviewer 2: ```suggestion
In the focus groups, the participants were asked about their experience of teachers leaving school during their early career induction. They were also shown the new prototype design and asked to use it to tell us that one of the teachers was leaving the school.
```

With the question about whether the teacher was moving to another school in England or not, around half the participants said they would not know the answer to this. The other half said they would often know, such as if they had to give a reference to the teacher’s school.

One participant commented that they might know the teacher was moving to another school in England, but not which one, and that the question wording was unclear on what to do in this case.

One participant commented that they would expect to be shown a category of reasons why a teacher was leaving the school.

One participant commented that they would like to be able to specify an expected return date for a teacher leaving temporarily.

### Wider issues

The research also highlighted that the process of a teacher leaving their school could be quite long, involving many steps with lots of people and organisations to inform.

> Telling the Department for Education about it within our service is often the very last step – if they even remember to do so.

Reviewer 2: ```suggestion
Using our service to tell the Department for Education is often the very last step – if they even remember to do so.
```

Some participants said they would expect the provider of the early career training to tell us instead.

Participants also mentioned that there is a separate process within [ECT Manager](http://ectmanager.com/) (which most schools use) to say that a teacher has left their school. This platform also lets them transfer the data about the teacher’s training (including their assessments) to the new school.

The requirement to use so many different platforms to enter the same (or similar) information was a cause of great frustration. It could also lead to confusion where the information was inconsistent across platforms.

## Next steps

> We plan to do some further work to better understand the implications of a teacher being temporarily absent on their early career training – for example, at what point they should be deferred or withdrawn.

Reviewer 2: ```suggestion
We plan to do some further work to better understand the implications of a teacher being temporarily absent from their early career training – for example, at what point they should be deferred or withdrawn.
```
